ranger/README.md

```diff
@@ -0,0 +1,29 @@
+# Apache Ranger
+
+This initialization action installs [Apache Ranger](https://ranger.apache.org/) on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+ Apache Ranger enables monitoring and managing data security across the Hadoop ecosystem and uses [Apache Solr](http://lucene.apache.org/solr/) for audits.
+
+## Using this initialization action
+
+You can use this initialization action to create a new Dataproc cluster with Apache Ranger installed:
+
+1. Use the `gcloud` command to create a new cluster with this initialization action. 
+The following command will create a new high availability cluster named `<CLUSTER_NAME>`.
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+      --num-masters 3 \
+      --initialization-actions gs://dataproc-initialization-actions/solr/solr.sh,\
+    gs://dataproc-initialization-actions/ranger/ranger.sh \
+      --image-version 1.2
+    ```
+1. Once the cluster has been created Apache Ranger Policy Manager should be running on **m-0** and use Solr in SolrCloud mode for audits.
+1. The Policy Manager Web UI is served on port 6080. 
```

Comment:
Can we make the port configurable?

Reply:
That should be possible I will make an optional flag to configure that.